x-pack/filebeat/module/okta/system/manifest.yml

```diff
@@ -32,6 +32,8 @@ var:
   - name: tags
     default: [forwarded]
   - name: url
+  - name: keep_state
+    default: false
```

Comment:
I think we would want to have this on by default to prevent users from hitting API rate-limiting on restarts. What was the reason for defaulting it to false.

Reply:
I agree that this should be ideally set to `true` by default. But this would change the way the module behave for people already using it. I think this would be considered a breaking change considering this was the expected previous behaviour. WDYT?
cc @jamiehynds since we also talked about this

Reply:
Well summed up Marc - I would love to see default as false but was afraid of a breaking change...even though it's probably a fixing change :/

Reply:
We have some flexibility with this still being beta. What affects will the users see if we set `keep_state: true` by default?

Reply:
They will follow from the last read event when restarting, and the first time they start the module, they will just pull last 24h instead of all the events if they do not set up another value for `initial_value`. This last bit could be addressed if we decide to set up a bigger value for it, though.

Reply:
Existing users will be less impacted by the 24h initial_interval since they should already have the older events indexed. Does the module use a stable `_id` value in docs to prevent duplicates? Assuming it does, then I doubt existing users will notice the change.

The docs you added around initial_interval are clear. So new users should be able to understand this well.

Reply:
In that case, I am unsure if having the `keep_state` toggle adds anything at all. Do you think could be just the way it behaves and expose `initial_interval` and that is it? This way this PR would be just a change in the docs.

Reply:
Removing `keep_state` SGTM.